Tasks/DockerV2/package-lock.json

```diff
@@ -67,22 +80,22 @@
       "resolved": "https://registry.npmjs.org/del/-/del-2.2.0.tgz",
       "integrity": "sha1-mlDwS/NzJeKDtPROmFM2wlJFa9U=",
       "requires": {
-        "globby": "4.1.0",
-        "is-path-cwd": "1.0.0",
-        "is-path-in-cwd": "1.0.1",
-        "object-assign": "4.1.1",
-        "pify": "2.3.0",
-        "pinkie-promise": "2.0.1",
-        "rimraf": "2.6.3"
+        "globby": "^4.0.0",
+        "is-path-cwd": "^1.0.0",
+        "is-path-in-cwd": "^1.0.0",
+        "object-assign": "^4.0.1",
+        "pify": "^2.0.0",
+        "pinkie-promise": "^2.0.0",
+        "rimraf": "^2.2.8"
```

Comment:
You may want to revert these changes or generate a package-lock.json without `^`s (compliance)

Reply:
+1, can use the changes in https://github.com/microsoft/azure-pipelines-tasks/pull/10569 to save the exact version

Reply:
--save-exact doesn't work on its own with npm > 6.*